### ISSUE TITLE
Questions about yolov5-p34.yaml

### ISSUE THREAD

**Issue by bzha5848** [ultralytics/yolov5#9675]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

Hi there,

I've got some questions about yolov5-p34.yaml. Just wondering compared to the traditional yolov5, why it was designed like that? And in parctise in what case can we use it? And is there any paper preseting this head design? I need to reference it in my thesis

Thanks so much!!

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@bzha5848 this model omits the P5 output. If you are only detecting very small objects this model may be suitable for your use case.

**Comment by bzha5848** [comment]
Yeah my object is super small, like the leaf diseases

**Comment by bzha5848** [comment]
Do you have any paper describing this structure? I need some reference for that

**Comment by Hehezcr** [comment]
> @bzha5848 this model omits the P5 output. If you are only detecting very small objects this model may be suitable for your use case.

Because my datasets are all single-target small targets and the anchor box size is small too, I used YOLOv5s-p34.yaml for training, and the final result found that YOLOv5s-p34 is nearly 4% higher than YOLOv5s' Recall, Precision, and Map, but I don't know why it is high. Since YOLOv5s only removes P5, shouldn't it just reduce the size of the model? How can it improve so much accuracy. My training set is 2600 and more than 40000 labels, and my validation set is more than 300 and 6000 labels. Hope for an explanation.

**Comment by glenn-jocher** [comment]
@Hehezcr P34 models do not have a P5 output. If your dataset consists of small objects only then this should work better, as your results show.
